realm-transformer/src/main/java/io/realm/transformer/RealmAnalytics.java

```diff
@@ -68,6 +68,7 @@
             + "      \"Anonymized MAC Address\": \"%USER_ID%\",\n"
             + "      \"Anonymized Bundle ID\": \"%APP_ID%\",\n"
             + "      \"Binding\": \"java\",\n"
+            + "      \"Uses Kotlin\": \"%USES_KOTLIN%\",\n"
```

Comment:
OK, so assuming the code works as it does now, do you see a syntax that’s cleaner than what I suggested above? I’m still happy with what I suggested, even if it means most of the time it will return nothing (undefined), and sometimes will report only Kotlin even if other languages are being used. Maybe over time we’ll have better ways to detect Java & Groovy I mean it will require some context to interpret the data but it’s cleaner than a bunch of top-level boolean properties for every possible JVM language if our detection ability improves in the future.